TCAT/Controllers/HomeViewController.swift

```diff
@@ -386,7 +386,7 @@ class HomeViewController: UIViewController, UITableViewDelegate, UITableViewData
         navigationItem.rightBarButtonItem = nil
         //Crashlytics Answers
         Answers.searchBarTappedInHome()
-        RegisterSession.shared?.logEvent(event: SearchBarTappedEventPayload(location: .home).toEvent())
+        let _ = RegisterSession.shared?.logEvent(event: SearchBarTappedEventPayload(location: .home).toEvent())
```

Comment:
is the `let _` if you don't use the value?

Reply:
Yeah, there are warnings if you don't have it. I don't know much about utilizing Register, have been meaning to dig into it